src/env.cr

```diff
@@ -52,6 +52,29 @@ module ENV
     !!LibC.getenv(key)
   end
 
+  # Fetch an environment variable corresponding to a key. Throws an exception
+  # if the key does not exist. If a second argument or a block is provided then
+  # this value is returned instead of throwing the exception.
+  #
+  def self.fetch(key)
+    fetch(key) do
+      if block = @@block
```

Comment:
Yes, `@block` is just an instance variable. It should be the hash's default proc if existent.